lib/keycloak/volumes/logging.properties

```diff
@@ -0,0 +1,70 @@
+#
```

Comment:
these are runtime configs not necessarily volumes. please see https://docs.docker.com/engine/reference/commandline/config/